cmake/modules/Buildpmdk.cmake

```diff
@@ -48,7 +53,7 @@ function(build_pmdk)
   set_target_properties(pmdk::pmem PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES ${PMDK_INCLUDE}
     IMPORTED_LOCATION "${PMDK_LIB}/libpmem.a"
-    INTERFACE_LINK_LIBRARIES Threads::Threads)
+    INTERFACE_LINK_LIBRARIES "${LIBPMEM_INTERFACE_LINK_LIBRARIES}")
```

Comment:
`LIBPMEM_INTERFACE_LINK_LIBRARIES` is a list, right?  What would quotes do here?

Reply:
yes, it is a list. adding quotes make it "`Threads::Threads;ndctl::ndctl;daxctl::daxctl`". I refer to other `buildxxx.make` and https://cmake.org/cmake/help/latest/prop_tgt/INTERFACE_LINK_LIBRARIES.html. Can I use list directly and better?


[target_link_libraries()](https://cmake.org/cmake/help/latest/command/target_link_libraries.html#command:target_link_libraries) will wrap each entry with the form ::@(directory-id);...;::@, where the ::@ is literal and the (directory-id) is unspecified.